fastlane/Fastfile

```diff
@@ -117,12 +118,68 @@ platform :ios do
       itc_provider: "33YMRSYD2L"
     )
 
-    # commit_version_bump(
-    #   message: "Version Bump v_#{udpated_version_number}_#{udpated_build_number} 🌳"
-    # )
+    tag_and_push(
+      version_number: udpated_version_number,
+      build_number: udpated_build_number
+    )
+
+  end
+
+  desc 'This lane commits the version number update and creates a PR if needed'
+  private_lane :branch_commit_push_and_create_pr_if_needed do |options|
+
+    version_number = options[:version_number]
+    build_number = options[:build_number]
+
+    git_status_output = sh("git status --porcelain ../#{pbx_project_file}")
+
+    git_diff_output = sh("git diff --unified=0 HEAD -- ../#{pbx_project_file}")
+
+    marketing_version_build_setting_pattern = /^\+\s+MARKETING_VERSION = (.+);/
+  
+    has_marketing_version_changes = git_diff_output.match?(marketing_version_build_setting_pattern)  
+
+    matching_required_pbx_changes = !git_status_output.empty? && has_marketing_version_changes
+
+    if matching_required_pbx_changes
+
+      version_bump_branch_name = "release-#{options[:version_number]}-build-#{build_number}-project-bump"
+
+      # Checkout a new branch
+      sh("git checkout -b #{version_bump_branch_name}")
+
+      # Commit the version bump only
+      git_commit(
+        path: "./#{pbx_project_file}", 
+        message: "Version Bump v_#{version_number}_#{build_number} 🌳"
+      )
+      
+      # Pushes the new branch with committed changes to Git
+      push_to_git_remote
+
+      # Creates a pull request from the current (newly created) branch against the main branch
+      create_pull_request(
+        api_token: ENV["GITHUB_ACCESS_TOKEN"],
```

Comment:
I guess we should look into using a "git bot" user instead of Jörn's as a follow-up of this change. I know there are a couple used in ecosia/core, such as EcosiaGitOpsBot, maybe we can even use it 🤔

Reply:
Yeah, I agree that it's not ideal that we have this ENV variable being backed by my account. I would be up to changing this to a bot user - also how we checkout `core` Let me know if you need assistance with this.

Reply:
Yes, agree to separate scopes ✂️